teamA/sangyeop/db/migrations/20220804020856_create_userPost_table.sql

```diff
@@ -0,0 +1,16 @@
+-- migrate:up
+CREATE TABLE posts (
+    id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    userId VARCHAR(100) NOT NULL,
+    userProfileImage VARCHAR(2000) NULL,
+    postingId VARCHAR(1000) NOT NULL,
+    postingImageUrl VARCHAR(2000) NULL,
+    postingContent VARCHAR(1000) NULL,
```

Comment:
테이블의 컬림 명을 만드실 때에는 camelCase가 아니라 snake_case를 사용해서 만들어주세요.
추가로 이미 post table의 컬럼이기 때문에, posting 이름을 없어도 될 것 같습니다.

```suggestion
    image_url VARCHAR(2000) NULL,
    content VARCHAR(1000) NULL,
```